### PR TITLE
Build bump

### PR DESCRIPTION
**Pull request by pierrsc**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pierrsc** [comment]
@conda-forge/staged-recipes please rerender

**Comment by chrisburr** [comment]
Can we get the bot to do it instead? 😄 

@conda-forge-admin, please rerender

**Comment by pierrsc** [comment]
@mariusvniekerk, the SQLAlchemy-migrate recipe never built correctly after you merged it to master. This PR is a "bump" to get that going again, and is ready for review/merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,12 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 8cfada30bbcf79a83b3a2d1bcd637de7e1f43e390d54de33e4f3249fd9aacfae
 
 build:
   number: 0
```
Comment (pierrsc) [inline recipe/meta.yaml:13]: 0.12.0, build 0 does not exist, so I did not bump it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,12 +8,12 @@ package:
...
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   build:
```
Comment (chrisburr) [inline recipe/meta.yaml]:
```suggestion
  host:
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -9,12 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by pierrsc** [comment]
@mariusvniekerk, this recipe is ready for your review.

**Comment by rluria14** [comment]
@pierrsc @mariusvniekerk @chrisburr @conda-forge/sqlalchemy-migrate please merge when you have a chance! thanks!

**Comment by beckermr** [comment]
On feedstocks, you need to bump the maintainers, @conda-forge/sqlalchemy-migrate-feedstock. The staged-recipes maintainers do not have commit rights here.

**Comment by ocefpaf** [comment]
BTW, the solve maintainer here is on vacation, so consider adding yourself as a maintainer in the future ;-p

**Pull request merged**